CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
+## [3.3.6] - 05/10/21
+### Added
+- Added Unit tests for BitmapUtils [#230](https://github.com/CanHub/Android-Image-Cropper/issues/230)
+
```

Comment:
In this case we do not need changelog updates, cause this update don't affect the final user. 
Even that I think is one of the most important parts

Reply:
Removing it

Reply:
Done